presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -428,6 +428,16 @@ public void testCast()
         assertQuery("SELECT CAST(true as VARCHAR), CAST(false as VARCHAR)");
         assertQuery("SELECT CAST(0.0 as VARCHAR)");
 
+        // Cast to varchar(n).
+        assertQuery("SELECT CAST('string_longer_than_1_character' AS VARCHAR(1)");
```

Comment:
Some of the test queries are on constant string arguments. Such CAST expressions get evaluated at the co-ordinator itself in the constant folding optimizer rule. Please add tests that are on column references here.

Reply:
Yes, you're right. I'm removing the test cases on constant string, as I have included test cases on column references of orders. They're redundant.